Makefile

```diff
@@ -19,9 +19,14 @@ default:
 		$(MAKE) deps
 		$(MAKE) all
 deps:
-		bash -c "godep restore"
+		bash -c "./scripts/deps.sh"
+
 test:
```

Comment:
The `test` block should either call both test (unit and integration) or be removed.